front/components/RenderMarkdown.tsx

```diff
@@ -149,17 +162,11 @@ function CodeBlock({
   children?: React.ReactNode;
 }): JSX.Element {
   const match = /language-(\w+)/.exec(className || "");
-  let language = match ? match[1] : "text";
-
-  const supportedLanguages = ["js", "ts", "tsx", "json", "text", "python"];
-  if (!supportedLanguages.includes(language)) {
-    language = "text";
-  }
-
+  const language = match ? match[1] : "text";
   const slate900 = slate["900"];
   const slate50 = slate["50"];
   const emerald500 = emerald["500"];
-  const amber500 = amber["500"];
+  const amber500 = amber["400"];
```

Comment:
?

Reply:
It was prettier with 400, I added a commit to rename the var to match the color. Thanks for catching!